luci-app-wireguard: add QR Code support plus fixes

dibdot opened the pull request:
* add optional QR Code support to status page (per wg Interface),
  submitted information via QR code are:
  - [Interface] (list of) Address / PrivateKey
  - [Peer] Endpoint / PublicKey / (list of) AllowedIPs
  - support multiple interfaces and peers
* fix XHTML rendering errors with OpenWrt standard theme
* fix logical glitch with empty descriptions, remove needless variables
* left align the output for better viewing esp. with material theme
* freshen up design a little bit
* switch space indentation to tabs

Signed-off-by: Dirk Brenken <dev@brenken.org>

dibdot commented:
A few screenshots from the "new" status page wir QR Codes:

**Default view (with collapsed QR Code):**
![wg_4](https://user-images.githubusercontent.com/8061346/46272962-c30a7500-c553-11e8-8505-682eb375b83a.png)

**QR Code view (of course usually without U2 tag ...;-)):**
![wg2](https://user-images.githubusercontent.com/8061346/46272989-e9301500-c553-11e8-914b-9f54cd48a31d.png)

**Hint for users without qrencode library:**
![wg3](https://user-images.githubusercontent.com/8061346/46273015-0cf35b00-c554-11e8-883c-c4ae05a59d26.png)

danrl commented:
I looked briefly over the proposed changes and they look good. I do not have access to a system where I could test this with. That means, the value of my review is limited.

Two minor questions:
* Would we need to escape the string that we send to `qrencode` somehow or can we trust it to not contain characters that would end the quotation of the string etc.?
* Do you mind saying a word about why you replaced `fieldset` by `div`. I think I can learn something here. Thank you!

danrl commented:
CC @zx2c4 
Look at this, the interface is getting cooler every day!

dibdot commented:
@danrl thanks for your feedback. Meanwhile I've added the interface address(es) to the provided QR Codes well - as requested by @nerdoc in #2180.
Regarding your questions:
- the qrcode input characters are coming from wg dump, therefore I would say they are safe. During my tests with the wireguard android app I didn't see any failures caused by unquoted chars or something like that.
- in the past "fieldset" was used for grouping & the "legend" tag - nowadays page structuring and grouping will be done by divs (we've removed the old table structure) and replaced legend with (a more screenreader friendly) "h3" tag

zx2c4 commented:
This is super cool. Thanks for pointing it out to be @danrl. It's a bit hard to review, though, because it appears that every line has changed, due to formatting fixes?

dibdot commented:
@danrl ok to merge?

danrl commented:
@dibdot It is fine for me, but as I said, I currently lack the setup and hardware to properly test it.

dibdot commented:
OK, than let's merge it to get more feedback! ;-)

shelterx commented:
Is it possible to add more data to the qrcode, at least the IP range and perhaps set the router ip as dns?
